extensions/shared/external-media/sources/with-media.js

```diff
@@ -53,6 +54,45 @@ export default function withMedia() {
 				};
 			}
 
+			componentDidMount() {
+				this.removeArrowKeysPropagationHandler = this.attachArrowKeysPropagationHandler();
+			}
+
+			componentWillUnmount() {
+				if ( this.removeArrowKeysPropagationHandler ) {
+					this.removeArrowKeysPropagationHandler();
+				}
+			}
+
+			attachArrowKeysPropagationHandler = () => {
+				const eventListenerEl = document.querySelector( '.components-modal__content' );
```

Comment:
Is there a `ref` or something we can utilize instead, or is this the only way to get the `el`?

Reply:
There's no ref forwarded to the Modal, but that's a good point and I'm going to check if we can use our own content wrapper to attach the listener, so we don't rely on external class name.